### ISSUE TITLE
npm install fails on version 2.0.1

### ISSUE THREAD

**Issue by BukeMan** [intuit/oauth-jsclient#68]
Latest version fails to install using `npm install` command. 

shell output:
```
npm WARN qbo-token-refresh-function@1.0.0 No repository field.
npm WARN qbo-token-refresh-function@1.0.0 No license field.

npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path C:\Users\xxxxx\source\repos\qbo-token-refresh-function\node_modules\intuit-oauth\bin\coveralls.js
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, chmod 'C:\Users\xxxxx\source\repos\qbo-token-refresh-function\node_modules\intuit-oauth\bin\coveralls.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\xxxxx\AppData\Roaming\npm-cache\_logs\2020-01-03T13_47_49_964Z-debug.log
```

**Comment by abisalehalliprasan** [comment]
@BukeMan : Kindly use the version [2.0.0](https://github.com/intuit/oauth-jsclient/tree/2.0.0) meanwhile. I am fixing the issue right now. 
Thanks.

**Comment by abisalehalliprasan** [comment]
@BukeMan : Its fixed with the new release [2.0.2](https://github.com/intuit/oauth-jsclient/tree/2.0.2)

Thanks for bringing it to our attention.
